apps_download/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017-2018 BizzAppDev
+# Copyright 2017 AgentERP
```

Comment:
Add Elico Corp
Dates -> 2019

Reply:
I don't see commit history for Elico. Do you mean yourself as reviewer? If so, it should go on contributors readme section, not here.

Reply:
@elicoidal  was contributed functionality for deciding the functionality as well as testing it, and also review.

Reply:
OK, I see it's in authors and in contributors,so I think it's enough

Reply:
Yes (thanks @bizzappdev ): I did not contribute via PR but in the overall functional design of the APPS store via meeting and project management.
It is a bit unusual but I think I deserve authorship for all modules in this repository

Reply:
You already had authorship, but it's no so important the copyright line in this file, so it's done.